ui/app/components/usage/page/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@ember/component';
+import { service } from '@ember/service';
+import type ApiService from 'vault/services/api';
+import type { getUsageDataFunction, UsageDashboardData } from '@hashicorp/vault-reporting/types/index';
```

Comment:
nit: can you group the type imports together below? https://github.com/hashicorp/vault/blob/main/ui/app/components/clients/date-range.ts#L11-L15

Reply:
Have grouped the type imports.